app/src/main/kotlin/com/mapzen/erasermap/view/MainActivity.kt

```diff
@@ -65,6 +67,8 @@ public class MainActivity : AppCompatActivity(), MainViewController, Router.Call
       @Inject set
     var crashReportService: CrashReportService? = null
       @Inject set
+    var apiKeys: ManifestModel? = null
+      @Inject set
```

Comment:
Do we need to inject an empty `ManifestModel` here when a new instance is created by `ManifestDownloader`?